supergraph.graphql

```diff
@@ -1,134 +0,0 @@
-
```

Comment:
@martinbonnin this is autogenerated but useful when just browsing the demo repo, suggest we leave this in.

Reply:
Oooopsie... very sorry, I didn't mean to commit this. Will rollback. Thanks for the heads up!